packages/atomic/src/components/ipx/atomic-ipx-modal/atomic-ipx-modal.tsx

```diff
@@ -33,6 +33,8 @@ export class AtomicIPXModal implements InitializableComponent<AnyBindings> {
   @State() public error!: Error;
 
   @Prop({mutable: true}) source?: HTMLElement;
+
+  @Prop() isEmbedded = false;
```

Comment:
I'm not sure how I feel about having the embedded option in a component titled modal, unless it looks like a modal.

Could you give me some examples of cases where this would be used?

Reply:
Maybe rename the modal for "window" 🤔